physical/consul.go

```diff
@@ -75,9 +76,10 @@ func newConsulBackend(conf map[string]string) (Backend, error) {
 
 	// Setup the backend
 	c := &ConsulBackend{
-		path:   path,
-		client: client,
-		kv:     client.KV(),
+		path:       path,
+		client:     client,
+		kv:         client.KV(),
+		permitPool: NewPermitPool(64),
```

Comment:
Can we make this a const like `DefaultParallelConsulOperations`? I think we can safely go to 128 as well. We should consider making this a tunable via the config, like a "max_parallel" flag.